Cubical/Codata/Conat/Properties.agda

```diff
@@ -0,0 +1,151 @@
+{- Conatural number properties (Tesla Ice Zhang et al., Feb. 2019)
+
+This file defines operations and properties on conatural numbers:
+
+- Infinity (∞).
+
+- Proof that ∞ + 1 is equivalent to ∞.
+
+- Proof that conatural is an hSet.
+
+- Bisimulation on conatural
+
+- Proof that bisimulation is equivalent to equivalence (Coinductive Proof
+  Principle).
+
+- Proof that this bisimulation is an hProp
```

Comment:
I would state this as "bisimulation is prop valued" instead

Reply:
Fixed.